js/plugins/vertexai/src/anthropic.ts

```diff
@@ -42,6 +42,11 @@ import {
   modelRef,
 } from '@genkit-ai/ai/model';
 import { GENKIT_CLIENT_HEADER } from '@genkit-ai/core';
+import z from 'zod';
+
+export const AnthropicConfigSchema = GenerationCommonConfigSchema.extend({
+  locationOverride: z.string().optional(),
```

Comment:
I'd just call it `location`

Reply:
I thought about that... had a mild preference to making it explicit... but `location` is shorter... therefore better! :)
Done.